Makefile

```diff
@@ -708,7 +708,16 @@ cpplint:
 
 ifneq ("","$(wildcard tools/eslint/bin/eslint.js)")
 lint: jslint cpplint
+CONFLICT_RE=^>>>>>>> [0-9A-Fa-f]+|^<<<<<<< [A-Za-z]+
 lint-ci: jslint-ci cpplint
+	@if [ "$(shell grep -IEqrs "$(CONFLICT_RE)" benchmark deps doc lib src test tools > /dev/null 2>&1; echo $$?)" != "0" \
```

Comment:
Can't this be simplified to `if ! ( grep -EIqrs ... ) && ! ( find . -maxdepth 1 ... ); then`?  You could swap the clauses, replace `&&` with `||` and drop the negation to shorten it even more.

You probably don't need to redirect stdio either if you are suppressing errors with -s.